crates/cli/src/subcommands/publish.rs

```diff
@@ -128,13 +112,9 @@ pub async fn exec(mut config: Config, args: &ArgMatches) -> Result<(), anyhow::E
         ));
     }
 
-    if trace_log {
-        query_params.push(("trace_log", "true"));
-    }
-
     let path_to_wasm = if !path_to_project.is_dir() && path_to_project.extension().map_or(false, |ext| ext == "wasm") {
```

Comment:
@bfops When is the proper time to remove this? Post 0.12?

Reply:
honestly probably 0.12. I'll see if I can revive the PR that removes this. I think multiple subcommands do this iirc.